pkg/scalers/ibmmq_scaler.go

```diff
@@ -178,19 +215,18 @@ func (s *IBMMQScaler) getQueueDepthViaHTTP(ctx context.Context) (int64, error) {
 	}
 	req.Header.Set("ibm-mq-rest-csrf-token", "value")
 	req.Header.Set("Content-Type", "application/json")
-	req.SetBasicAuth(s.metadata.username, s.metadata.password)
 
-	client := kedautil.CreateHTTPClient(s.defaultHTTPTimeout, s.metadata.tlsDisabled)
+	req.SetBasicAuth(s.metadata.username, s.metadata.password)
```

Comment:
Do still need this basic auth using mTLS?

Reply:
Yes, it still need basic auth. We also asked this question at IBM itself. IBM's answer is that for now you cannot use certificates to replace username/password, that would require modifying te webuser.xml file which is not currently supported on the MQ Appliance.
Also the mention that although this functionality is not currently available in the MQ Appliance, they will evaluating adding it into a future release.

Reply:
Quite weird mTLS approach 🤣 
nvm, it's okey, thanks for the info!